src/App.js

```diff
@@ -2,17 +2,38 @@ import React from "react";
 import Form from "./form";
 
 function App() {
-  const [nameCompany, setNameCompany] = React.useState("Able");
+  const [nameCompany, setNameCompany] = React.useState(null);
+  const [nameCEO, setNameCEO] = React.useState(null);
 
-  return (
-    <>
+  function saveName(name) {
```

Comment:
I must recognize that this new function is truly necessary

Reply:
yeah, it's really helpful